<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DuplicatedObjectsHandlerDlg</class>
 <widget class="QDialog" name="DuplicatedObjectsHandlerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duplicated Objects Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOverride">
       <property name="text">
        <string>Override</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCreateCopies">
       <property name="text">
        <string>Create Copies</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DuplicatedObjectsHandlerDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
